src/docker-images/init-container/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:16.04 as builder
+
+RUN apt-get update && apt-get install -y wget gzip build-essential
+
+WORKDIR /ssh_build
+
+COPY build-openssh-static.sh /ssh_build
+RUN sh build-openssh-static.sh
+
+#FROM python:3.8.0-alpine3.10
+FROM ubuntu:16.04
```

Comment:
Does SSH build in 16.04 work for other versions of Ubuntu?

Reply:
Yes, have it tested in 16.04 and 18.04.